### PR TITLE
fix(S3): content review and update

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Object Storage documentation pages revision and correction.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: faq/coldstorage.mdx [hunk 1]
```diff
@@ -56,7 +56,7 @@ We recommend that you use Glacier to archive data that you may need only once or
   The time it takes to restore an object depends on the size of the object and on if [multipart](/storage/object/concepts/#multipart-uploads) is configured. It can take anywhere from a few minutes to 24 hours for restore to start (for objects larger than 1 MB). 
   Unfortunately, we are unable to provide estimates of how long data retrieval takes for each use case.
 
-  To facilitate the steps of restoration and ensure a fast restitution of your data, we recommend that you use average sized files (larger than 1 MB).
+  To facilitate the steps of restoration and ensure a fast restitution of your data, we recommend you use average-sized files (larger than 1 MB).
 </Message>
 
 Find out more about [how to restore an object from glacier](/storage/object/how-to/restore-an-object-from-glacier) in our dedicated page.
```
Comment (nerda-codes) [inline faq/coldstorage.mdx:62]:
```suggestion
Find out [how to restore an object from glacier](/storage/object/how-to/restore-an-object-from-glacier) in our dedicated page.
```

File: storage/object/api-cli/bucket-operations.mdx [hunk 2]
```diff
@@ -41,11 +41,11 @@ aws s3 mb s3://bucketname
...
+  Deleting a bucket will also delete all of its objects. For safety measures, the effective deletion of the bucket will start 24 hours after the deletion request. The bucket name will then be available for other customers to use.<br /><br />The billing of your resources stops as soon as you send the delete request.<br /><br />Once you do so, the bucket and its objects and versions will no longer be displayed in the console or available via the APIs. For this reason, we recommend you erase all objects and versions in the bucket before requesting the deletion, if you wish the bucket deletion to be effective immediately.
```
Comment (nerda-codes) [inline storage/object/api-cli/bucket-operations.mdx:44]:
```suggestion
  Deleting a bucket will also delete all of its objects. For safety measures, the effective deletion of the bucket will start 24 hours after the deletion request. The bucket name will then be available for other customers to use.<br /><br />The billing of your resources stops as soon as you request the deletion of your resources.<br /><br />Once you do so, the bucket and its objects and versions will no longer be displayed in the console or available via the API. For this reason, we recommend you delete all objects and versions in the bucket before requesting the deletion, if you wish the bucket deletion to be effective immediately.
```

File: storage/object/how-to/monitor-consumption.mdx [hunk 3]
```diff
@@ -25,15 +25,15 @@ You can view your Object Storage's metrics and logs by using [Scaleway Cockpit](
 
 ## How to activate your Cockpit
 
-Scaleway Cockpit is deactivated by default, but you can easily activate it in the Scaleway console by following [this procedure](/observability/cockpit/how-to/activate-cockpit/).
+Scaleway Cockpit is deactivated by default, but you can [follow our documentation](/observability/cockpit/how-to/activate-cockpit/) to activate with a few steps.
```
Comment (nerda-codes) [inline storage/object/how-to/monitor-consumption.mdx:28]:
```suggestion
Scaleway Cockpit is deactivated by default, but you can [read our documentation](/observability/cockpit/how-to/activate-cockpit/) to activate it in a few steps.
```

File: storage/object/how-to/monitor-consumption.mdx [hunk 4]
```diff
@@ -25,15 +25,15 @@ You can view your Object Storage's metrics and logs by using [Scaleway Cockpit](
...
 
-To view your Object Storage metrics and logs, use the Grafana dashboard which is accessible from the Scaleway console:
-  - First you must create a user and their associated credentials (a username and password) for Grafana
-  - Then use these credentials to access the Grafana dashboard for all your Scaleway products, via the Scaleway console.
+To view your Object Storage metrics and logs, use the Grafana dashboard, which is accessible from the Scaleway console:
+  - First you must create a user and their associated credentials (a username and password) for Grafana.
+  - Then use these credentials to access the Grafana dashboard for all your Scaleway products via the Scaleway console.
 
-1. Create a Grafana user and retrieve their username and password by following [this procedure](/observability/cockpit/how-to/retrieve-grafana-credentials/).
+1. Create a Grafana user and [retrieve their username and password](/observability/cockpit/how-to/retrieve-grafana-credentials/).
```
Comment (nerda-codes) [inline storage/object/how-to/monitor-consumption.mdx:36]:
```suggestion
1. Create a Grafana user and [retrieve your username and password](/observability/cockpit/how-to/retrieve-grafana-credentials/).
```

File: faq/coldstorage.mdx [hunk 5]
```diff
@@ -56,10 +56,10 @@ We recommend that you use Glacier to archive data that you may need only once or
   The time it takes to restore an object depends on the size of the object and on if [multipart](/storage/object/concepts/#multipart-uploads) is configured. It can take anywhere from a few minutes to 24 hours for restore to start (for objects larger than 1 MB). 
   Unfortunately, we are unable to provide estimates of how long data retrieval takes for each use case.
 
-  To facilitate the steps of restoration and ensure a fast restitution of your data, we recommend that you use average sized files (larger than 1 MB).
+  To facilitate the steps of restoration and ensure a fast restitution of your data, we recommend you use average-sized files (larger than 1 MB).
 </Message>
 
-Find out more about [how to restore an object from glacier](/storage/object/how-to/restore-an-object-from-glacier) in our dedicated page.
+Find out [how to restore an object from glacier](/storage/object/how-to/restore-an-object-from-glacier) in our dedicated page.
```
Comment (RoRoJ) [inline faq/coldstorage.mdx:62]: Elsewhere, Glacier is often capitalized - to check?

File: storage/object/api-cli/generate-aws4-auth-signature.mdx [hunk 6]
```diff
@@ -40,7 +40,7 @@ SignedHeaders
 Signature
 : A signed hash consisting of a hash of the request body, your secret key, and information about the request (i.e., the canonical request). |
 
-The canonical request included in the signature is made up of:
+The canonical request included in the signature is made of:
```
Comment (RoRoJ) [inline storage/object/api-cli/generate-aws4-auth-signature.mdx:43]: I think the original version is more correct?

File: storage/object/how-to/use-obj-stor-with-private-networks.mdx [hunk 7]
```diff
@@ -44,18 +44,18 @@ Follow the instructions for [creating an Instance](/compute/instances/how-to/cre
 
 ## How to create a Private Network and attach the Instance
 
-1. Follow the instructions for [creating a Private Network](/network/vpc/how-to/create-private-network/). Make sure you create it in the region that encompasses the Availability Zone of the Instance you previously created.
+1. Follow the instructions for [creating a Private Network](/network/vpc/how-to/create-private-network/). Make sure you create it in the same region of the Availability Zone of the Instance you previously created.
```
Comment (RoRoJ) [inline storage/object/how-to/use-obj-stor-with-private-networks.mdx:47]: I think I find the original clearer, but maybe it's personal preference?  Or perhaps "...in the region that contains/includes the Availability Zone" ?

File: faq/coldstorage.mdx (hunk 5 above)
Comment (jcirinosclwy) [inline faq/coldstorage.mdx:62]:
```suggestion
Find out [how to restore an object from Glacier](/storage/object/how-to/restore-an-object-from-glacier) in our dedicated page.
```

File: storage/object/api-cli/generate-aws4-auth-signature.mdx (hunk 6 above)
Comment (jcirinosclwy) [inline storage/object/api-cli/generate-aws4-auth-signature.mdx:43]:
```suggestion
The canonical request included in the signature is made up of:
```

File: storage/object/how-to/use-obj-stor-with-private-networks.mdx (hunk 7 above)
Comment (jcirinosclwy) [inline storage/object/how-to/use-obj-stor-with-private-networks.mdx:47]:
```suggestion
1. Follow the instructions for [creating a Private Network](/network/vpc/how-to/create-private-network/). Make sure you create it in the region that encompasses the Availability Zone of the Instance you previously created.
```

File: storage/object/api-cli/object-storage-aws-cli.mdx [hunk 8]
```diff
@@ -29,7 +29,7 @@ The AWS-CLI is an open-source tool built on top of the AWS SDK for Python (Boto)
 
 ## How to install the AWS-CLI
 
-The AWS-CLI is an open-source tool built on top of the AWS SDK for Python (Boto) that provides commands for interacting with AWS services. With minimal configuration, you can start using all the functionality provided by the AWS Management.
+The AWS-CLI is an open-source tool built on top of the AWS SDK for Python (Boto) that provides commands for interacting with AWS services. With minimal configuration, you can start using all the functionality provided by AWS Management.
 
 To interact with Object Storage, `aws-cli` and `awscli-plugin-endpoint` need to be installed. The `awscli-plugin-endpoint` is a great plugin to help people more easily access third-party S3 providers.
```
Comment (nerda-codes) [inline storage/object/api-cli/object-storage-aws-cli.mdx:34]:
```suggestion
To interact with Object Storage, `aws-cli` and `awscli-plugin-endpoint` need to be installed. The `awscli-plugin-endpoint` is a great plugin to easily access third-party S3 providers.
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**